pkg/execution/svccatadmission/internaldns.go

```diff
@@ -18,6 +18,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func isInternalDNSServiceClass(serviceInstance sc_v1b1.ServiceInstance) bool {
```

Comment:
Pass a pointer here. By conventions API type objects are always passed by pointer. And it makes sense - they may be big.

Reply:
done!